ci/azure-test-stable.yml

```diff
@@ -41,6 +41,10 @@ jobs:
     - script: cargo check --benches
       displayName: Check benchmarks
 
-  # Run with no default defaults
-  - script: cargo ${{ parameters.cmd }}
-    displayName: cargo ${{ parameters.cmd }} --no-default-features
+  # Run with all features
+  - script: cargo ${{ parameters.cmd }} --all-features
+    displayName: cargo ${{ parameters.cmd }} --all-features
+
+  # Run with no default features
+  - script: cargo check --no-default-features
+    displayName: cargo check --no-default-features
```

Comment:
Should still remain `${{ parameters.cmd }}`?

Reply:
[I changed it to check only](https://github.com/tokio-rs/bytes/pull/293/commits/bbf7785c722266d566a01d7fb8a099cb44e3bef5) because some (many?) tests seemed to depend on the std feature.

Reply:
I think we can change this to `${{ parameters.cmd }}`, but as bytes doesn't change the behavior between std and no_std, check-only should be fine.

(the actual script that is executed on CI is `script:`, not `displayName:`, so the current master has not tested/checked with no-std.)

https://github.com/tokio-rs/bytes/blob/97c13388331c6e2948278d73d3644b07a3946dda/ci/azure-test-stable.yml#L45-L46

Reply:
Ah yeah indeed.